### PR TITLE
fix(instructions): react-hooks 02.01

### PR DESCRIPTION
**Pull request by MikeRatcliffe**
Exercise `react-hooks/exercises/02.side-effects/01.problem.effects/index.tsx`

The current instructions don't make sense... this patch clears things up.

### CODE REVIEW

**Comment by MikeRatcliffe** [comment]
Oops, it's supposed to be broken :/

**Comment by kentcdodds** [comment]
We can still clear things up a bit by explaining that it's supposed to be broken inline if you think that would provide clarity

File: exercises/02.side-effects/01.problem.effects/index.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ function App() {
 
 	// 🐨 add a useEffect(() => {}, []) call here (we'll talk about that empty array later)
 	// 🐨 in the useEffect callback, subscribe to window's popstate event
+	// 🐨 if that doesn't make sense to you... don't worry, it's supposed to be broken!
```
Comment (kentcdodds) [inline exercises/02.side-effects/01.problem.effects/index.tsx:22]:
```suggestion
	// 🦉 if that doesn't make sense to you... don't worry, it's supposed to be broken! We'll fix it next
```

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**